public/pages/intro-class.html

```diff
@@ -237,14 +211,14 @@ <h4>Software Engineering Career Beyond Refcode</h4>
           <p>Engineers who have reached the Engineer III level will be encouraged to look for roles outside of Refcode. Engineers should be eligible to get a job at a company with salaries in the range of $45,000/year to $80,000/year, depending on experience, motivation, and skill.</p>
         </div>
         <p>Sign up for the Summer 2023 cohort now!</p>
-        <p><a href="https://internal.refcode.org/waitlist"class="button mobile-only">Apply Now</a></p>
+        <p><a href="https://internal.refcode.org/login"class="button mobile-only">Apply Now</a></p>
         <p><a href="https://internal.refcode.org/volunteer" class="button">Volunteer</a></p>
       </div>
       <aside class="floater">
         <p><strong>10 week class</strong> - $200 (scholarships available for refugees and immigrants)</p>
-        <p><strong>June 28-September 6 </strong>, Wednesday nights, 6-9pm</p>
+        <p><strong>August 2-October 4 </strong>, Wednesday nights, 6-9pm</p>
         <p>Students, sign up for the Summer 2023 cohort now!</p>
-        <p><a href="https://internal.refcode.org/waitlist" class="button">Apply Now</a></p>
+        <p><a href="https://internal.refcode.org/login" class="button">Apply Now</a></p>
```

Comment:
We should provide more instructions on what they need to do to apply. If they log in, they'll be confused.

Reply:
<h2>Applying for the Class:</h2>
        <p>To apply for the upcoming cohort, follow these steps:</p>
        <ol>
          <li>Click on Apply Now</li>
          <li>Create an account</li>
          <li>Log in using the credentials you used in the previous step</li>
          <li>Go to Web Development 1 assessment</li> 
          <li>Watch the welcome video</li>
          <li>Compelete the tasks </li>
          <li>Come to Refcode to get your assessment reviewed and approved</li>
        </ol>

Reply:
Did you see this ? I included the steps to enroll